### PR TITLE
fantiger project added

### PR DESCRIPTION
**Pull request by agrawalia**

### CODE REVIEW

**Comment by 0xBoxer** [comment]
@philippWassibauer

File: pages/fantiger/fantiger.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+# List of relevant dashboard slugs prefixed with username
+dashboards:
+  - "fantiger/dashboard"
+
+# List of relevant query-ids.
+queries:
+  - 2672702
+  - 2672698
+  - 2672614
+
```
Comment (philippWassibauer) [inline pages/fantiger/fantiger.md:44]:
```suggestion
---
```

**Review by philippWassibauer — APPROVED** [review]

**Comment by philippWassibauer** [comment]
the build didn't run as the SVG is more than 1MB (it is over 2MB). @agrawalia can you reduce the size?

**Comment by philippWassibauer** [comment]
I reverted the PR, so please resubmit. thanks!

**Comment by agrawalia** [comment]
@philippWassibauer,
I have resubmitted PR https://github.com/duneanalytics/projects/pull/194 , please approve

**Pull request merged**